Sources/SwiftInspectorCommands/Tests/TypeLocationCommandSpec.swift

```diff
@@ -0,0 +1,187 @@
+              expect(result?.didFail) == true
+            }
+          }
+        }
+
+        context("name is valid") {
+          context("with no --path argument") {
+            it("fails") {
+              let result = try? TestTypeLocationTask.run(path: nil, name: "Foo")
+              expect(result?.didFail) == true
+            }
+          }
+
+          context("with an empty --path argument") {
+            it("fails") {
+              let result = try? TestTypeLocationTask.run(path: "", name: "Foo")
+              expect(result?.didFail) == true
+            }
+          }
+        }
+
+        context("when path is a directory") {
+          beforeEach { pathURL = try? Temporary.makeFolder() }
+
+          it("fails") {
+            let result = try? TestTypeLocationTask.run(path: pathURL.path, name: "Foo")
+            expect(result?.didFail) == true
+          }
+        }
+
+        context("when path is a file") {
+          beforeEach { pathURL = try? Temporary.makeFile(content: "struct Foo { }") }
+
+          it("succeeds") {
+            let result = try? TestTypeLocationTask.run(path: pathURL.path, name: "Foo")
+            expect(result?.didSucceed) == true
+          }
+
+          context("type is found") {
+            it("outputs the correct line numbers") {
+              let contents =
+              """
+              import Foundation
+
+              struct Foo { }
+              """
+
+              pathURL = try? Temporary.makeFile(content: contents)
+
+              let result = try? TestTypeLocationTask.run(path: pathURL.path, name: "Foo")
+              expect(result?.outputMessage).to(contain("2 2"))
+            }
+          }
+
+          context("type is not found") {
+            it("outputs nothing") {
+              let contents =
+              """
+              import Foundation
+
+              struct Foo { }
+              """
+
+              pathURL = try? Temporary.makeFile(content: contents)
+
+              let result = try? TestTypeLocationTask.run(path: pathURL.path, name: "Bar")
+              expect(result?.outputMessage).to(beEmpty())
+            }
+          }
+
+          context("multiple types found") {
+            it("outputs line numbers for each type") {
+              let contents =
+              """
+              import Foundation
+
+              struct Foo { }
+
+              public final class Bar {
+                enum Foo { }
+              }
+              """
+
+              pathURL = try? Temporary.makeFile(content: contents)
+
+              let result = try? TestTypeLocationTask.run(path: pathURL.path, name: "Foo")
+              let lines = result?.outputMessage?.split { $0.isNewline }
+              expect(lines?.count) == 2
+              expect(lines?.first).to(equal("2 2"))
+              expect(lines?.last).to(equal("5 5"))
+            }
+          }
+        }
+      }
+
+      describe("outputString") {
```

Comment:
I don't feel strongly. I removed https://github.com/fdiaz/SwiftInspector/pull/13/commits/51041bc608658908c65bdab35dbe1210f0333d66

@fdiaz can you help me understand how it ties us to ArgumentParser though? I guess because I'm instantiating the command directly?